src/main/java/org/takes/facets/fork/FkRegex.java

```diff
@@ -193,12 +198,27 @@ public TkRegex get() {
     public FkRegex(final Pattern ptn, final Scalar<TkRegex> tke) {
         this.pattern = ptn;
         this.target = tke;
+        this.removeslash = true;
+    }
+
+    /**
+     * Allows disabling the standard way for handling trailing slashes.
+     * @param enabled Enables/Disables the removal of a trailing slash.
+     * @return FkRegex
+     */
+    public FkRegex setRemoveTrailingSlash(final boolean enabled) {
+        this.removeslash = enabled;
+        return this;
     }
 
     @Override
     public Opt<Response> route(final Request req) throws Exception {
         String path = new RqHref.Base(req).href().path();
-        if (path.length() > 1 && path.charAt(path.length() - 1) == '/') {
+        if (
+                this.removeslash
+                && path.length() > 1
+                && path.charAt(path.length() - 1) == '/'
```

Comment:
@mreiche Wrong indentation here as well. Please, use 4 spaces always.

Reply:
How to solve?
```
Checkstyle: C:\projects\takes\src\main\java\org\takes\facets\fork\FkRegex.java[219]: 'if' child has incorrect indentation level 12, expected level should be 16. (IndentationCheck)
```

Reply:
Incorrect indentation level. Please change from 12 to 16.